airbyte-integrations/connectors/source-shopify/source_shopify/spec.json

```diff
@@ -76,6 +65,20 @@
             }
           }
         ]
+      },
+      "shop": {
```

Comment:
Since shop is required for oauth let's put it before auth cc @misteryeo

Reply:
Done, it will be: Shop, Authentication Method, Start Date.